convert_hf_to_gguf.py

```diff
@@ -663,7 +663,11 @@ def get_vocab_base_pre(self, tokenizer) -> str:
             res = "minerva-7b"
         if chkhsh == "8b5a93ed704057481f240da0be7e7dca721d7f8f4755263b6807227a2cbeae65":
             # ref: https://huggingface.co/sentence-transformers/stsb-roberta-base
-            res = "roberta-bpe"
+            # NOTE: The Roberta tokenizer is the same as GPT-2, but it always
+            #   adds the cls/sep tokens as bos/eos. This is handled as a
+            #   post-processor in tokenizers, so the chkhsh is different, but
+            #   it still maps to gpt-2 internally.
+            res = "gpt-2"
```

Comment:
as per guidelines, we shouldnt be modifying this value in convert_to_gguf , so that it can be autogenerated from convert_hf_to_gguf_update.py

we want it to map to gpt-2 tokenization type. 

Would the correct way to do this be keep it to roberta-bpe as generated and then add it to mapping https://github.com/ggerganov/llama.cpp/blob/master/src/llama.cpp#L6483 here , so that it maps to gpt-2?

Any input would be appreciated

Reply:
> Would the correct way to do this be keep it to roberta-bpe as generated and then add it to mapping master/src/llama.cpp#L6483 here , so that it maps to gpt-2?

Yes, this is the correct way.

Reply:
@ggerganov Thank you! I have pushed the change and tested it.

I dont think the failing test in this PR is because of my changes? I see other PRs failing in same place